src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+$cell-size: 100px;
+$gap: 1px;
+$digit-size: 0.8 * $cell-size;
+$shift: $cell-size / 2;
+$week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$cell-count: 7;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  width: ($cell-size + $gap) * $cell-count;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    position: relative;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid black;
+    padding: 10px;
+
+    @for $count from 1 through 31 {
+      &:nth-child(#{$count}n)::before {
+        content: '#{$count}';
+
+        width: $digit-size;
+        height: $digit-size;
+
+        position: absolute;
+
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
```

Comment:
```suggestion
```

Reply:
Consider moving this style to a general class instead of generating the same styling in the cycle for every 31 element